corehq/form_processor/models.py

```diff
@@ -258,9 +259,7 @@ class AbstractAttachment(models.Model):
 
     @property
     def filepath(self):
-        if getattr(settings, 'IS_TRAVIS', False):
-            return os.path.join('/home/travis/', str(self.attachment_id))
-        return os.path.join('/tmp/', str(self.attachment_id))
+        return os.path.join(gettempdir(), str(self.attachment_id))
```

Comment:
Hard-coded paths here seems wrong. [`gettempdir()`](https://docs.python.org/2/library/tempfile.html#tempfile.gettempdir) should provide a platform-independent solution, although I'm not sure what it will return when running on travis. Hence temporary modification to `.travis.yml`.